File: apps/problems/max-element/boilerplate-full/function.js
//##USER_CODE_HERE##
  
  const input = require('fs').readFileSync('/dev/problems/max-element/tests/inputs/##INPUT_FILE_INDEX##.txt', 'utf8').trim().split('\n').join(' ').split(' ');
  const size_arr = parseInt(input.shift() || "0");
      const arr = input.splice(0, size_arr).map(x => parseInt(x));
  const result = maxElement(arr);
  console.log(result);
    

File: apps/problems/max-element/boilerplate-full/function.cpp
#include <iostream>
#include <vector>
#include <string>
#include <sstream>
#include <climits>



//##USER_CODE_HERE


int main() {
    int m_arr;
std::cin >> m_arr;
std::vector<int> arr(m_arr);

for(int i = 0; i < m_arr; ++i) {
    std::cin >> arr[i];
}
    Solution obj;
    int result = obj.maxElement(arr);
    std::cout << result << std::endl;
    return 0;
}
